TriangleApp/app/src/main/java/triangle/triangleapp/helpers/MediaHelper.java

```diff
@@ -0,0 +1,192 @@
+package triangle.triangleapp.helpers;
+
+import android.hardware.Camera;
+import android.media.CamcorderProfile;
+import android.media.MediaRecorder;
+import android.os.Environment;
+import android.util.Log;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import triangle.triangleapp.view.CameraPreview;
+
+/**
+ * Created by danie on 9-6-2017.
+ */
+
+public class MediaHelper {
+    private final String TAG = "MediaHelper";
+    private boolean isRecording;
+    private MediaRecorder mMediaRecorder;
+    private CameraPreview mCameraPreview;
+    private Camera mCamera;
+    private final int MEDIA_TYPE_IMAGE = 1;
+    private final int MEDIA_TYPE_VIDEO = 2;
+    private WebSocket webSocket;
+    private String url;
+    private String protocol;
+    private int mediaRecorderMaxDuration;
+
+    public MediaHelper(CameraPreview cameraPreview) {
+        mCameraPreview = cameraPreview;
+        isRecording = false;
+        mediaRecorderMaxDuration = 5000;
+        url = "ws://145.49.35.215:1234/send";
+        protocol = "ws";
+        webSocket = new WebSocket(url, protocol);
+    }
+
+    public void record(){
+        if (isRecording) {
+            isRecording = false;
+            stopStreaming(true);
+        } else {
+            startStreaming(true);
+        }
+    }
+
+    private void initializeVideoRecorder(boolean firstInit) {
```

Comment:
Added comment to function initializeVideoRecorder(boolean firstInit).
/**
* Initializing mediarecorder, set settings and send stream via web socket.
* @param firstInit, Check initializing on first time.
*/